translations/wesnoth-A_New_Order/cs.po

```diff
@@ -23033,71 +22951,73 @@ msgstr ""
 
 #. [then]
 #: A_New_Order/scenarios/10_Siege_of_Haeltin.cfg:280
-#, fuzzy
-#| msgid "We won't pay."
 msgid "We don't have any oxen."
-msgstr "Nic vám nedáme."
+msgstr "Nemáme žádné voly."
 
 #. [then]
 #: A_New_Order/scenarios/10_Siege_of_Haeltin.cfg:281
 msgid "Then something similar."
-msgstr ""
+msgstr "Tak něco podobného."
 
 #. [then]
 #: A_New_Order/scenarios/10_Siege_of_Haeltin.cfg:298
 msgid ""
 "...like that boar! Let me go capture it, sacrifice it, and then surely God's "
 "favor will be upon us!"
 msgstr ""
+"... jako je tenhle kanec! Nech mě to chytit, obětovat, a pak jistě bude nad "
```

Comment:
```suggestion
"... jako je tenhle kanec! Nech mě to chytit, obětovat a pak jistě bude nad "
```

Reply:
So are disagreements about what we refer to as the "Oxford comma" (in English) a thing in Czech, too? Personally I like the "Oxford comma", but if the consensus in Czech is that it's bad there, then fine.

Reply:
In Czech we have a rule (not consensus) for using comma in lists. Generally a comma in Czech have a power to change meaning of whole sentence, so rules are here to avoid ambiguity.